scrutiny.json

```diff
@@ -0,0 +1,82 @@
+{
+    "Scrutiny": {
+        "containers": {
+            "scrutiny": {
+                "image": "ghcr.io/linuxserver/scrutiny",
+                "launch_order": 1,
+                "opts": [
+                    [
+                        "--cap-add",
+                        "SYS_RAWIO"
+                    ],
+                    [
+                        "--cap-add",
+                        "SYS_RAWIO"
+                    ],
+                    [
+                        "--device",
+                        "/dev:/dev"
+                    ],
+                    [
+                        "-v",
+                        "/run/udev:/run/udev:ro"
+                    ],
+                    [
+                        "--privileged",
+                        ""
+                    ]
+                ],
+                "ports" : {
+                    "8080": {
+                        "description": "Port for scrutiny's web interface and API.",
+                        "host_default": 8080,
+                        "label": "WebUI HTTP port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Scrutiny config files. Eg: create a Share called Scrutiny-Share-Config for this purpose alone.",
+                        "label": "Config share"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid system UserID for Scrutiny. Must have full permissions on the previous Share.",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid system GroupID for Scrutiny. This group or the above UserID must have full permissions on the previous Share.",
+                        "label": "GID",
+                        "index": 2
+                    },
+                    "SCRUTINY_API_ENDPOINT": {
```

Comment:
I assume, the port on the API_ENDPOINT needs to be the same as the one set up in the ports section? If so, you want to add this to the tooltip, that the port needs to be the same as defined above (as default, and of course, the entire thing could be different if used as a remote collector).

Reply:
The official docker deployment for this application is made out of multiple containers, so for `analogj/scrutiny:collector` it make sense to specify the API_ENDPOINT of `analogj/scrutiny:web` because they are not in the same container. But as in linuxserver image the collector and web are in the same "host", it make sense to have localhost:8080. 
In this case, I preferred to leave it optional and allow the user to chose in case he have another collector deployed elsewhere. That's why one can also change `SCRUTINY_COLLECTOR` to false.